pkgs/by-name/wi/wizard-engine/package.nix

```
@@ -0,0 +1,78 @@
+{
+  fetchFromGitHub,
+  jdk,
+  lib,
+  makeWrapper,
+  stdenv,
+  versionCheckHook,
+  virgil3,
+  which,
+}:
+
+stdenv.mkDerivation {
+  pname = "wizard-engine";
+  version = "25β.2881";
```

Can we avoid the special beta character in the version number? That will surely break something.


```suggestion
  version = "25b.2881";
```